cli/test/abis/valid/IInterpreterV1.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
this is not an abi, its a compilation artifact that _includes_ the abi as one of the keys

also, is there no way to have these built locally and referenced rather than committing many thousands of lines of code to the repo?

Reply:
@rouzwelt not committing them to the repo, is there a way they can be automatically built locally, e.g. with a `build.rs` file or similar?

Reply:
hmm, there should be a way ofc, not sure where to source those data from though, maybe we can use the built artifacts of the sol files within the same repo